sdk/src/main/java/com/sygic/travel/sdk/di/KodeinSetup.kt

```diff
@@ -0,0 +1,76 @@
+package com.sygic.travel.sdk.di
+
+import android.content.Context
+import com.github.salomonbrys.kodein.Kodein
+import com.github.salomonbrys.kodein.bind
+import com.github.salomonbrys.kodein.instance
+import com.github.salomonbrys.kodein.singleton
+import com.github.salomonbrys.kodein.with
+import com.google.gson.Gson
+import com.google.gson.GsonBuilder
+import com.sygic.travel.sdk.BuildConfig
+import com.sygic.travel.sdk.auth.api.AuthApiClient
+import com.sygic.travel.sdk.auth.facade.AuthFacade
+import com.sygic.travel.sdk.auth.service.AuthService
+import okhttp3.Cache
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+
+object KodeinSetup {
+
+	fun setupKodein(context: Context) = Kodein {
+		bind<Context>() with singleton { context }
+		import(constants)
+		import(networkModule)
+		import(authNetworkModule)
+		import(authorizationModule)
+	}
+
+	private val constants = Kodein.Module {
+		constant("auth_endpoint") with "https://tripomatic-auth-master-testing.sygic.com"
```

Comment:
I would inline these constants for now, the actual url will have to be also configurable, let's change this later.

Reply:
:heavy_check_mark: